public/admin.php

```diff
@@ -101,6 +102,62 @@
     return back()->withErrors(__('legacy.error.error'));
 }
 
+if ($action === 'alt_identifier') {
+    $altsForUser = request()->input('u');
+
+    $forUser = User::where('User', $altsForUser)->first();
+    if ($forUser == null) {
+        $message = "Unknown user: $altsForUser";
+    } else {
+        $altsForUser = $forUser->User;
+
+        $emailAddresses = [];
+        if (!empty($forUser->EmailAddress)) {
+            $emailAddresses[] = $forUser->EmailAddress;
+        }
+        if (!empty($forUser->email_backup && $forUser->email_backup != $forUser->EmailAddress)) {
```

Comment:
I seem to be missing this `email_backup` column in my `UserAccounts` table, but I'm wondering if there is potentially a logic error here due to a typo:

```suggestion
        if (!empty($forUser->email_backup) && $forUser->email_backup != $forUser->EmailAddress) {
```